### ISSUE TITLE
[Bug]: comments column not read properly

### ISSUE THREAD

**Issue by rdemaria** [pylhc/tfs#114]
### Documentation

- [X] Yes

### Operating System

linux

### Python Version

3.9

### Package Version

3.4.0

### Bug Description

The column comments should be string in the table attached

[o.zip](https://github.com/pylhc/tfs/files/10920157/o.zip)

### Steps to Reproduce

tfs.read(fn)

### Relevant output

```shell
In [11]: t['COMMENTS']
Out[11]: 
0        NaN
1        NaN
2        NaN
3        NaN
4        NaN
        ... 
17825    NaN
17826    NaN
17827    NaN
17828    NaN
17829    NaN
Name: COMMENTS, Length: 17830, dtype: object
```
```


### Possible Fix Implementation

_No response_

**Comment by fsoubelet** [comment]
Thanks for providing a file, I've done a few tests with it. Considering other string columns are read as expected, I assume there is an issue when reading empty strings (`""`).

After a few tests on a dummy file I am quite certain that the behavior comes from `pandas`. Reading the simple `csv` file below with the same settings as in `tfs-pandas` parses the empty strings as `NaN`s just as in your example.

| File | Reading result |
| :--: | :------------: |
|![Screenshot 2023-03-08 at 13 41 57](https://user-images.githubusercontent.com/19598248/223728357-c55a84fd-1f99-4f50-b079-44b65f7703c8.png)|<img width="314" alt="Screenshot 2023-03-08 at 13 41 51" src="https://user-images.githubusercontent.com/19598248/223728458-b6c93ca7-2d34-4ce3-bc71-7a42284f4784.png">|

I suggest for now we go to `pandas` maintainers themselves to see if they consider it intended behavior or an issue.

As a workaround I guess for now you can get back to empty strings with:
```python
df.select_dtypes(include=['string','object']).fillna("", inplace=True)
```

**Comment by rdemaria** [comment]
In addition, this triggers the validators that pollute the stdout with a very long warning.

**Comment by fsoubelet** [comment]
Fixed by #116. Fix is included in v3.5.0
